### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by brooksmtownsend**
This is gonna be fun

### CODE REVIEW

File: host_core/lib/host_core/application.ex [hunk 1]
```diff
@@ -22,8 +22,7 @@ defmodule HostCore.Application do
     # for other strategies and supported options
     opts = [strategy: :one_for_one, name: HostCore.Supervisor]
 
-    IO.puts "Starting Host Core"
+    IO.puts("Starting Host Core")
```
[the remark this answers is not in the document]
Comment (brooksmtownsend) [reply host_core/lib/host_core/application.ex]: Fixed, will just need a linux-built httpserver binary to pass the tests 😄

[the remark this answers is not in the document]
Comment (autodidaddict) [reply host_core/lib/host_core/application.ex]: That's checked into the invocation pr.

[the remark this answers is not in the document]
Comment (brooksmtownsend) [reply host_core/lib/host_core/application.ex]: Once #27 is merged, I'll rebase this branch off of that one. Once tests pass I can merge this in, we can deal with the specifics of a GH action iteratively, I'd rather us be able to see the test output for future features

**Comment by autodidaddict** [comment]
#27 has been merged @brooksmtownsend , what's the status of this PR?

**Comment by brooksmtownsend** [comment]
@autodidaddict this has been rebased, tests pass but I'd just like to get the dialyzer to run before we merge. I'll let you know when it's working

**Pull request merged**